themes/bootstrap3/templates/RecordDriver/DefaultRecord/versions-link.phtml

```diff
@@ -0,0 +1,21 @@
+<?php
+  $versions = $this->driver->tryMethod('getOtherVersionCount');
+  if ($versions):
+?>
+  <?php
+    if (isset($this->tabs['Versions'])) {
+      $url = !empty($this->full) ? $this->escapeHtmlAttr($this->recordLink()->getTabUrl($this->driver, 'Versions')) . '#tabnav' : '#versions';
```

Comment:
Full link is used when linking from e.g. results list to the Versions tab. The short link is used for activating the correct tab without reloading the whole page.
I believe AJAX tabs are always enabled unless you don't use AJAX. Then the link wouldn't work, but is it really necessary to keep the compatibility at that level? I could attach some JS that opens the tab instead of the full link, but I'd rather avoid that if possible.

Reply:
Okay, that makes sense, and I agree it's not worth going crazy for non-JS support here... just wanted the confirmation that I wasn't forgetting something. I'm happy to leave this as-is!